### ISSUE TITLE
[Feature Request] - Include workspace name in page title for better UX

### ISSUE THREAD

**Issue by brolifen** [kasmtech/workspaces-issues#376]
When you use something for the very first time you tend to notice things that might be improved more quickly. I noticed if I had multiple tabs open with workspaces that I had to go through all of them to find the one I needed because the tabs are all called "Kasm Worspaces". This user experience could be easily improved if the title of the page matched the workspace name.

![image](https://user-images.githubusercontent.com/20615514/236612259-310ed28d-9248-4d24-b7b5-0268ff6dd435.png)

**Comment by SnoutBug** [comment]
Yes, please. Something like this - especially if you use multiple apps - is really handy.
Also for automation with, lets say, KeePass; autotype may rely on the window title for different input combinations

**Comment by teja-kasm** [comment]
We’re currently enhancing our GitHub issue tracker to improve organization and efficiency.

As part of this process, we’re closing older issues to help keep the tracker focused on current and active topics.

If this issue is still relevant or you’re still experiencing the problem, please feel free to create a new issue.

Thank you for your understanding!

**Comment by SnoutBug** [comment]
@teja-kasm This issue is still relevant as nobody has worked on it.
Opening a new issue is counter productive.
